### PR TITLE
Add circleci file for neomad

### PR DESCRIPTION
**Pull request by Ydrasil**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+machine:
+  python:
+    version: 3.5.0
```
Comment (vinyll) [inline circle.yml:3]: Can't you just tell 3.5 instead of 3.5.0 (we want it to work with the latest version).
Eventually also 3.6

**Comment by vinyll** [comment]
ok, let's merge this and improve later…

File: circle.yml (hunk 1 above)
Comment (Ydrasil) [inline circle.yml]: So we have to change this line into `version: 3` ?

**Comment by vinyll** [comment]
We won't need it for now, we're on py3.5 all the way 👍

**Pull request merged**